java/common/src/test/scala/org/typelevel/otel4s/java/ContextStorageSuite.scala

```diff
@@ -0,0 +1,43 @@
+package org.typelevel.otel4s.java
+
+import cats.effect.IO
+import cats.effect.IOLocal
+import cats.effect.Resource
+import cats.effect.std.Dispatcher
+import io.opentelemetry.context.Context
+import io.opentelemetry.context.ContextKey
+import io.opentelemetry.context.ContextStorage
+import java.util.logging._
+
+object Run extends cats.effect.IOApp.Simple {
+
+  val key = ContextKey.named[String]("test")
+
+  def run =
+    Dispatcher.parallel[IO].use { dispatcher =>
+      for {
+        _ <- IO {
+          val rootLog = Logger.getLogger("")
+          rootLog.setLevel(Level.FINE)
+          rootLog.getHandlers().head.setLevel(Level.FINE)
+        }
+        ioLocal <- IOLocal(null: Context)
```

Comment:
We have to add the wrapper before we refer to `Context`, or else we get a FINE message that we're being ignored.